src/docs/development/tools/devtools/memory.md

```diff
@@ -5,41 +5,68 @@ description: Learn how to use the DevTools memory view.
 
 ## What is it?
 
-Allocated Dart objects created using a class constructor
-(for example, by using `new MyClass()` or `MyClass()`)
-live in a portion of memory called the heap.
+Allocated Dart objects created using a class constructor (for
+example, by using `new MyClass()` or `MyClass()`) live in a partion
+of memory called the heap that is managed by the Dart VM (virtual
+machine). 
 
 ## DevTools memory page
```

Comment:
delete this header and move the following info under what is it

Reply:
Please review I've updated with the latest UX and re-organized the doc to have a better flow.